application/tests/test_auth.py

```diff
@@ -0,0 +1,97 @@
+import json
+from .tests import BaseTest
```

Comment:
You could use a more descriptive name for the module that has the BaseTest class. For example, `test_setup`, `base_test` etc. `tests` sounds too general and gives the impression that all your tests are in that module.